db/migrate/20211109200705_backfill_type_on_vita_partner.rb

```diff
@@ -0,0 +1,15 @@
+class BackfillTypeOnVitaPartner < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def up
+    orgs = VitaPartner.where(parent_organization_id: nil)
```

Comment:
It can be bad form to use a model in a migration: https://jakeyesbeck.com/2021/04/10/avoid-models-in-migrations/

I suppose it doesn't matter in this case necessarily, so it's not a blocker. If you are willing to change it, that's excellent -- what do you think?

I encourage you to use raw SQL to 
`db/migrate/20201222174316_move_user_id_from_role_to_user.rb` is an example.